src/main/java/com/uid2/shared/attest/AttestationTokenRetriever.java

```diff
@@ -33,12 +34,11 @@ public class AttestationTokenRetriever {
     private final AtomicReference<String> attestationToken;
     private final AtomicReference<String> optOutJwt;
     private final AtomicReference<String> coreJwt;
-    private final Handler<Integer> responseWatcher;
+    private final Handler<Pair<Integer, String>> responseWatcher;
```

Comment:
Are you sure this is only used by operator?

Reply:
Out of the core, optout, admin, and operator, the attestation token retriever is used by operator and optout, but optout uses a null responseWatcher. Will look into optout closer to make sure it is not broken by these changes

Reply:
I have tested optout/core with my shared changes locally and it is able to attest and refresh with no extra changes needed